### PR TITLE
[PY3] Add unicode_literals to high-priority modules/states

### PR DESCRIPTION
**Pull request by terminalmage**

### CODE REVIEW

**Comment by rallytime** [comment]
@terminalmage There a some test failures here that are related. I think these ones are for sure related, the others might need another run to see if it's flakiness or related:

- integration.modules.test_cp.CPModuleTest.test_get_file_str_https
- integration.modules.test_cp.CPModuleTest.test_get_file_str_local
- integration.modules.test_cp.CPModuleTest.test_get_url_file_no_dest
- integration.modules.test_cp.CPModuleTest.test_get_url_https_no_dest

**Comment by terminalmage** [comment]
The cp module tests were failing for the same reason as some of the others, because of str/unicode mismatches. I added my same workaround for ``ModuleCase.run_function()`` that I added for the git module integration tests.

I could not reproduce the other failures, and did not touch the runners tests anyway.

**Comment by rallytime** [comment]
re-run py

File: tests/unit/modules/test_disk.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def test_tune(self):
                 self.assertEqual(len(args[0].split()), 6)
                 self.assertEqual(kwargs, {'python_shell': False})
 
-    def test_format(self):
+    def test_ormat(self):
```
Comment (rallytime) [inline tests/unit/modules/test_disk.py]: I think this was an accident to change the name of this test function.

**Review by rallytime — CHANGES_REQUESTED** [review]
Only one small change, otherwise this looks great.

**Comment by rallytime** [comment]
@terminalmage This has a merge conflict now, as well.

**Comment by terminalmage** [comment]
@rallytime This is fixed and rebased.

**Pull request merged**